src/main/java/repositories/TicketDAO.java

```diff
@@ -0,0 +1,88 @@
+package repositories;
+
+import models.Ticket;
+import models.TicketType;
+
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TicketDAO {
+
+    private Connection connection;
+
+    public TicketDAO(Connection connection) {
+        this.connection = connection;
+    }
+
+    public void addTicket(Ticket ticket){
+        String sqlCommand = "INSERT INTO ticket (id, user_id, ticket_type, creation_date) VALUE (?, ?, ?, ?)";
```

Comment:
perfectly will be to convert all queries (`"INSERT INTO ticket (id, user_id, ticket_type, creation_date) VALUE (?, ?, ?, ?)"`) in static variables to create them once with class instead of creating with each time of method call